src/framework/statusDropdown/_statusDropdown.scss

```diff
@@ -39,11 +43,11 @@
     background-color: $dropdownDotGreyLine;
   }
 
-  &:before {
+  &:not(&--disabled):before {
     transform: rotate(-45deg) translate(-2.5px, -2.5px);
   }
 
-  &:after {
+  &:not(&--disabled):after {
```

Comment:
&:not(&--disabled) {
  &:before, &:after {
  }
  &:before {
  }
 &:after {
  }
}

or you can also set the before and after ($dropdownArrow_width....) in a placeholder selector and extend it when you have the single selectors for before and after http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholders